python/src/worker/constants.py

```diff
@@ -0,0 +1,6 @@
+DOWNLOAD_EXPERIMENT = 'download_experiment_s3'
```

Comment:
Do we also need a FAILED_TASK message too?

Reply:
I don't think so, because failures are communicated differently I think.